modules/cnvkit/1.0/config/default.yaml

```diff
@@ -0,0 +1,64 @@
+lcr-modules:
+    
+    cnvkit:
+
+        # TODO: Update the list of available wildcards, if applicable
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "__UPDATE__"
+            sample_bai: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            hard_masked: True
+            target_bed:
+                agilent: ""
+                idt: ""
+            batch: "--drop-low-coverage "
+            SNPs: 
+                quality: 20
+                opts: "--ignore-RG  --skip-indels  --count-orphans  --annotate DP,AD "
+            BAF:
+                rescale: "clonal"
+                min_depth: 20
+            scatter:
+                min_depth: 20
+                ymax: 10
+                ymin: -10
+
+
+        conda_envs:
+            cnvkit: "{MODSDIR}/envs/cnvkit-0.9.7.yaml"
+            bcftools: "{MODSDIR}/envs/bcftools-1.10.2.yaml"
+            
+        threads:
+            batch: 24
+            seg: 8
+
+
+        resources:
+            batch: 
+                mem_mb: 10000
+            SNPs: 
+                mem_mb: 8000
+                bam: 1
+            seg: 
+                mem_mb: 6000
+            outputs:
+                mem_mb: 6000
+
+
+        pairing_config:
+            genome:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+            capture:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+            exome:
```

Comment:
I don't think this a supported seq_type in the lcr-modules. The variant callers include the exome and capture within only `capture` seq_type, can we do it here as well for consistency?

Reply:
Okay I removed "exome" from the module